ls1intum/Artemis
# Feature/text/enable text assessmnet v2

### Checklist

### Motivation and Context
We merged Text Assessment V2 some time ago in #1251, #1286, #1307. 
In this PR, we delete Text Assessment V1 and update the URL of V2 (remote `-new`).

### Steps for Testing

1. Create Text Submissions
2. Assess them
3. See that it opens the new UI (e.g. from Tutor Dashboard).
